fossunited/fossunited/doctype/event_free_ticket_applications/event_free_ticket_applications.py

```
@@ -23,6 +23,7 @@ class EventFreeTicketApplications(Document):
         event: DF.Link | None
         full_name: DF.Data
         organization: DF.Data | None
+        tshirt_size: DF.Literal["XS", "S", "M", "L", "XL", "2XL", "3XL"]
```

Ruff (0.16.1)

[error] Undefined name `XS`

(F821)